sysadmin/systemd/core/ipol-archive.service

```diff
@@ -3,7 +3,7 @@ Description=Run archive
 After=network.target
 
 [Service]
-ExecStart=/bin/bash -c "source ./venv/bin/activate && python main.py archive.conf"
+ExecStart=/bin/bash -c "source ./venv/bin/activate && uvicorn archive:app --host 0.0.0.0 --port 9000"
 WorkingDirectory=/home/ipol/ipolDevel/ipol_demo/modules/archive/
 Environment=IPOL_HOST=ipolcore.ipol.im
 Environment=IPOL_URL=https://ipolcore.ipol.im
```

Comment:
No problems if we hardcode this and it's run in a different environment?

Reply:
This service is specific to the `core` server, and not meant to be used elsewhere without modification.

Reply:
No, because the IPOL_URL is the same on all servers of a given deployment.

Reply:
OK!